README.md

```diff
@@ -43,7 +43,12 @@ your own repository.
 
 1. Edit `application.rb` and change the module name and configuration settings.
 2. Edit the database names in `database.yml`
-3. Search the project for the word `PROJECT_NAME` to find other things to change.
+3. Use "search and replace" to replace project specific strings across all files:
+   - PROJECT_DOMAIN => project-name.com
+   - PROJECT_NAME_PARAM => project-name
+   - PROJECT_NAME_PASCAL => ProjectName
+   - PROJECT_NAME_SNAKE => project_name
+   - PROJECT_NAME_HUMAN => Project name
```

Comment:
I'd have thought these would just be correctly cased versions of `PROJECT_NAME`.
As in `ProjectName`, `project-name`, `project_name` and `Project name` 🤔 
then one could use smart search/replace to convert all of them at once.
Any reason you did it this way?

Reply:
That was my initial idea, but then when I got to the domain, I got worried that two differently used strings could look the same but not necessary have to be the same.

That problem might be hard to spot now because I later decided to include the `.com`-part in the domain, but I think it's still a potential issue.

For instance at some point we might want to have a "PROJECT_NAME_HEROKU", it might often be "project-name" but it's not reliably the same.